packages/truss/src/generate.ts

```diff
@@ -118,6 +118,10 @@ class CssBuilder<T extends Properties> {
     return this.newCss({ selector: ":hover" });
   }
   
+  get scrollbarHidden() {
+    return this.addIn("&::-webkit-scrollbar", { display: "none"}).add("scrollbarWidth", "none");
```

Comment:
@stephenh - We don't currently have any `newMethod` functions that can do a `addIn`. I was thinking of building out one, but I need to do both `addIn` and `add`, which felt very bespoke. So instead I decided to just hard code this property into the generate file. What do you think?
(This hides scrollbars on scrollable containers. The user can still scroll, but the scrollbar isn't visible. We plan on adding gradients to indicate their is content overflow one way or the other.)

Reply:
Yep, seems fine to just hard-code a full/typed-out method here & there.

Do we have a way to add "just a random chunk of code" to the output w/o changing truss itself?

If so (or if not, maybe we add that here instead? like the extension mechanism), I wouldn't mind keeping this seems somewhat experimental (imo?) method inside of internal-frontend; just a little skeptical reinventing scrollbars via gradients, but admittedly I don't have any of the context.

Reply:
Yea, that might be a better idea. Maybe can add this to Beam's Truss config... 🤔